### DISCUSSION TITLE
Set language in your profile

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#1552] [Announcements]
Soon you will not have to switch language anymore, but you can set a preferred language in your profile. 

We will let logged-in users know that feature is there with a banner:

![image](https://user-images.githubusercontent.com/1003685/146244305-0e8cb639-cac5-45a7-8d8c-21d93c3c8b35.png)

You can set the language via your profile:

![image](https://user-images.githubusercontent.com/1003685/146244389-68f1a9f3-eb0d-4234-96b4-ca34e826b52e.png)

**Comment by das-g** [comment]
Now, when one is logged in, the language switcher isn't shown at the top right (or anywhere else, as far I can tell). That means, one has to go to the profile and change the preferred language setting to switch to another language (or log out, so that the language switcher is visible again).

I like to switch between languages I know to compare the content. Would it be possible to (maybe optionally, controlled by a default-off flag in the profile) show the language switcher also for logged in users?
